apps/passport-server/src/services/generic-issuance/pipelines/CSVPipeline/makeMessagePCD.ts

```diff
@@ -0,0 +1,40 @@
+import { MessagePCD, MessagePCDPackage } from "@pcd/message-pcd";
+import { ArgumentTypeName, SerializedPCD } from "@pcd/pcd-types";
+import { v4 as uuid } from "uuid";
+import { traced } from "../../../telemetryService";
+
+/**
+ * Given a CSV row uploaded by a user to a {@link CSVPipeline},
+ * generates an {@link MessagePCD} containing {@link }signed
+ * {@code process.env.GENERIC_ISSUANCE_EDDSA_PRIVATE_KEY}
+ *
+ *
+ * - `title`: gets displayed in the title of the PCD
+ * - `message`: markdown-formatted announcement. can include images, links, etc.
+ */
+export async function makeMessagePCD(
+  csvRow: string[],
+  eddsaPrivateKey: string
+): Promise<SerializedPCD> {
+  return traced("", "makeMarkdownPCD", async () => {
```

Comment:
```suggestion
  return traced("genericIssuanceService", "makeMarkdownPCD", async () => {
```

Reply:
i deliberately use an empty service name in for bare functions like this.